packages/excalidraw/change.ts

```diff
@@ -1450,7 +1450,9 @@ export class ElementsChange implements Change<SceneElementsMap> {
         continue;
       }
 
-      redrawTextBoundingBox(boundText, container, elements, false);
+      redrawTextBoundingBox(boundText, container, elements, {
+        informMutation: false,
```

Comment:
I don't recall, why is this not needed any more ?

Reply:
haha, I fell for the same. The `informMutation=false` is kept. Previously it was just a `boolean` flag.

Though through refactor the API signature change is no longer necessary so we can potential revert.

Reply:
reverted 🙂